src/ValueRepresentation.ts

```diff
@@ -144,12 +143,11 @@ export default class ValueRepresentation {
           } else if (this.maxLength) {
             valid = checklen <= this.maxLength;
           }
-
           var errmsg = "Value exceeds max length, vr: " + this.type + ", value: " + checkValue + ", length: " + displaylen;
           if (!valid) {
-            // if(isString)
-                // console.log(errmsg)
-            // else
+             if(isString)
+                 console.log(errmsg)
```

Comment:
Random `console.log` is typically unhelpful. Is there some action to take here?

Reply:
It will only print if the DICOM is malformed, which I think is useful enough to justify, but it's not a big deal either way.